src/pages/index.js

```diff
@@ -135,10 +136,158 @@ class Home extends Component {
+                  },
+                },
+              ]}>
+              <div
+                css={{
+                  display: 'flex',
+                  flexDirection: 'row',
+
+                  [media.lessThan('medium')]: {
+                    display: 'block',
+                    whiteSpace: 'nowrap',
+                  },
+                }}>
+                {marketingColumns.map((column, index) => (
+                  <div
+                    key={index}
+                    css={{
+                      display: 'flex',
+                      flexDirection: 'column',
+                      flex: '0 1 33%',
+                      marginLeft: 40,
+
+                      '&:first-of-type': {
+                        marginLeft: 0,
+
+                        [media.lessThan('medium')]: {
+                          marginLeft: 10,
+                        },
+                      },
+
+                      [media.lessThan('medium')]: {
+                        display: 'inline-block',
+                        verticalAlign: 'top',
+                        marginLeft: 0,
+                        whiteSpace: 'normal',
+                        width: '75%',
+                        marginRight: 20,
+                        paddingBottom: 40,
+
+                        '&:first-of-type': {
+                          marginTop: 0,
+                        },
+                      },
+
+                      '& p': {
+                        lineHeight: 1.7,
+                      },
+                    }}>
+                    <h3
+                      css={{
+                        '&&': {
+                          // Make specificity higher than the site-wide h3 styles.
+                          color: colors.subtle,
+                          marginBottom: 20,
+                          paddingTop: 0,
+                          fontWeight: 300,
+                          fontSize: 20,
+
+                          [media.greaterThan('xlarge')]: {
+                            fontSize: 24,
+                            fontWeight: 200,
+                          },
+                        },
+                      }}>
+                      {column.title}
+                    </h3>
+                    <div dangerouslySetInnerHTML={{__html: column.content}} />
+                  </div>
+                ))}
+              </div>
+            </section>
+            <hr
+              css={{
+                height: 1,
+                marginBottom: -1,
+                border: 'none',
+                borderBottom: `1 solid ${colors.divider}`,
+              }}
+            />
+            <section css={sectionStyles}>
+              <div id="examples">
+                <div className="example">
+                  <h3>A Simple Component</h3>
+                  <p>
+                    React components implement a `render()` method that takes
```

Comment:
The content for these sections also needs to come from markdown files (so it can be localized).